pages/[resourceType]/create.tsx

```diff
@@ -46,19 +51,84 @@ const CreateResourcePage = () => {
       <Center>
         <Button
           disabled={hasLintError}
-          onClick={editorSubmitHandler}
+          onClick={() => {
+            editorSubmitHandler(codeEditorContents);
+          }}
           color="cyan"
           variant="filled"
           size="lg"
+          radius="md"
         >
           Submit Resource
         </Button>
       </Center>
+      {resourceType === "Group" && (
+        <div style={{ textAlign: "center", marginTop: "30px", marginBottom: "20px" }}>
+          <Divider my="md" />
+          <div>
+            {" "}
+            <Text weight={600} size="xl" color={textGray} style={{ marginBottom: "20px" }}>
+              {" "}
+              OR{" "}
+            </Text>{" "}
+          </div>
+
+          <div style={{ margin: "auto", width: "85vh" }}>
+            <MantineProvider
+              // changes hex values associated with each Mantine color name to improve UI
+              theme={{
+                colors: {
+                  gray: replaceBackground,
+                },
+              }}
+            >
+              <MultiSelectComponent
+                resourceType="Patient"
+                setValue={setPatientValue}
+                value={patientValue}
+              />
+            </MantineProvider>
+          </div>
+          <Button
+            style={{ marginTop: "30px" }}
+            onClick={convertToJSON}
+            color="cyan"
+            variant="filled"
+            size="lg"
+            radius="md"
+          >
+            Create Group
+          </Button>
+        </div>
+      )}
     </div>
   );
 
+  //called when the create group button is clicked. Converts patient list into a group resource, then
+  //calls editorSubmitHandler to handle the POST request
+  function convertToJSON() {
+    const groupMember = patientValue.map((el) => {
```

Comment:
Can use more type safety here:

```suggestion
    const groupMember: fhirJson.GroupMember[] = patientValue.map((el) => {
```